indica/equilibrium.py

```diff
@@ -540,6 +543,62 @@ def enclosed_volume(
         volume = self.area.interp(rhop=rhop).interp(t=t)
         return volume, t
 
+    @classmethod
+    def read_geqdsk(cls, filename: Path) -> "Equilibrium":
```

Comment:
I would recommend to separate the reader form the class and add it to the /readers/ folder.
This is cleaner, you separate data reading from class instatiation, and you give the user the possibility to "just" read the eqdsk file and use if for something else. 

You will then need 2 lines to instatiate the Equilibrium class with this data, but it's worth it.

Reply:
Absolutely agree! Tried fitting it in to `DataReader` at first but figured it was too different to just read from a file, so it's just a standalone class